build/media-suite_helper.sh

```diff
@@ -399,7 +399,7 @@ do_wget() {
             ;;
         esac
 
-        if check_hash "$archive" "$hash"; then
+        $quiet || if check_hash "$archive" "$hash"; then
```

Comment:
Unsure if this line is needed or not

Reply:
Yeah, this is not working properly for 404s. And even worse, it's returning 0.

Reply:
I might just remove this until it's actually a problem

Reply:
Removed for now